include/tudat/astro/observation_models/observationModel.h

```diff
@@ -441,6 +477,45 @@ class ObservationModel
 
 protected:
 
+    void checkReferenceLinkEndForTimeBiases( const LinkEndType linkEndAssociatedWithTime ) const
+    {
+        std::vector< int > indexRefLinkEndTimeBias;
+        std::shared_ptr< observation_models::ObservationBias< ObservationSize > > biasCalculator = this->observationBiasCalculator_;
+        if ( std::dynamic_pointer_cast< observation_models::ConstantTimeBias< ObservationSize > >( biasCalculator ) != nullptr )
+        {
+            indexRefLinkEndTimeBias.push_back( std::dynamic_pointer_cast< observation_models::ConstantTimeBias< ObservationSize > >( biasCalculator )->getLinkEndIndexForTime( ) );
+        }
+        else if ( std::dynamic_pointer_cast< observation_models::ArcWiseTimeBias< ObservationSize > >( biasCalculator ) != nullptr )
+        {
+            indexRefLinkEndTimeBias.push_back( std::dynamic_pointer_cast< observation_models::ArcWiseTimeBias< ObservationSize > >( biasCalculator )->getLinkEndIndexForTime( ) );
+        }
+        else if ( std::dynamic_pointer_cast< observation_models::MultiTypeObservationBias< ObservationSize > >( biasCalculator ) != nullptr )
+        {
+            std::vector< std::shared_ptr< observation_models::ObservationBias< ObservationSize > > > biasList =
+                    std::dynamic_pointer_cast< observation_models::MultiTypeObservationBias< ObservationSize > >( biasCalculator )->getBiasList( );
+            for ( unsigned int j = 0 ; j < biasList.size( ) ; j++ )
+            {
+                if ( std::dynamic_pointer_cast< observation_models::ConstantTimeBias< ObservationSize > >( biasList[ j ] ) != nullptr )
+                {
+                    indexRefLinkEndTimeBias.push_back( std::dynamic_pointer_cast< observation_models::ConstantTimeBias< ObservationSize > >( biasList[ j ] )->getLinkEndIndexForTime( ) );
+                }
+                else if ( std::dynamic_pointer_cast< observation_models::ArcWiseTimeBias< ObservationSize > >( biasList[ j ] ) != nullptr )
+                {
+                    indexRefLinkEndTimeBias.push_back( std::dynamic_pointer_cast< observation_models::ArcWiseTimeBias< ObservationSize > >( biasList[ j ] )->getLinkEndIndexForTime( ) );
+                }
+            }
+        }
+
+        for ( unsigned int j = 0 ; j < indexRefLinkEndTimeBias.size( ) ; j++ )
+        {
+            if ( indexRefLinkEndTimeBias[ j ] != observation_models::getLinkEndIndicesForLinkEndTypeAtObservable(
+                    observableType_, linkEndAssociatedWithTime, linkEnds_.size( ) ).at( 0 ) )
+            {
+                throw std::runtime_error( "Error when setting time biases, ref link end is different from time ref link in observation simulation settings." );
```

Comment:
Why is an exception thrown if the reference link end is not the same as the link end with the time bias? If the two are not the same, should the link not simply be inactive?